lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -2460,7 +2538,8 @@ namespace Js
     }
 
     template <typename T>
-    void DictionaryTypeHandlerBase<T>::InvalidateFixedField(DynamicObject* instance, PropertyId propertyId, DictionaryPropertyDescriptor<T>* descriptor)
+    template <typename TPropertyKey>
```

Comment:
Ah, yes. Of course. :)

---
In reply to: [79732431](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r79732431) [](ancestors = 79732431,79702610)